realm/realm-library/build.gradle

```diff
@@ -65,6 +65,16 @@ android {
                 }
             }
         }
+
+        buildTypes {
+            debug {
+                javaCompileOptions {
+                    annotationProcessorOptions {
+                        arguments = [ debug : 'true' ]
```

Comment:
If `arguments` is a dictionary, I think that it's better to just add an entry instead of replacing dictionary itself.
I guess there is a chance that other plugins already setting their options.